torch/onnx/symbolic.py

```diff
@@ -1195,7 +1202,7 @@ def feature_dropout(g, input, p, train):
 feature_alpha_dropout_ = feature_alpha_dropout
 
 
-@parse_args('v', 't', 'i', 'i')
+@parse_args('v', 't', 'is', 'i')
```

Comment:
Why change this?

Reply:
Similarly to mean, [torch.norm](https://pytorch.org/docs/stable/torch.html#torch.norm) accepts tuple types.
Before this change, `torch.norm(d, dim=(1,2))` would have failed.

Reply:
if so, can we also add test cases?

Reply:
Similarly to mean, [torch.norm](https://pytorch.org/docs/stable/torch.html#torch.norm) accepts tuple types.
Before this change, torch.norm(d, dim=(1,2)) would have failed.

Reply:
added two tests.